src/storage/query/ScanVertexProcessor.cpp

```diff
@@ -63,9 +63,6 @@ void ScanVertexProcessor::doProcess(const cpp2::ScanVertexRequest& req) {
     RowReaderWrapper reader;
     for (int64_t rowCount = 0; iter->valid() && rowCount < rowLimit; iter->next()) {
         auto key = iter->key();
-        if (!NebulaKeyUtils::isVertex(spaceVidLen_, key)) {
```

Comment:
Remove it in ScanEdgeProcessor as well

Reply:
ScanEdgeProcessor  can handle in-edge， so `isEdge  `needs.